framework/wazuh/core/exception.py

```diff
@@ -208,7 +220,21 @@ class WazuhException(Exception):
                'remediation': 'Please, use GET /decoders/files to list all available decoders'
                },
         1504: {'message': 'The decoder does not exist or you do not have permission to see it',
-               'remediation': f'Please, visit the official documentation (https://documentation.wazuh.com/'
+               'remediation': f'Please, visit the official documentation '
+                              f'(https://documentation.wazuh.com/'
+                              f'{DOCU_VERSION}/user-manual/reference/ossec-conf/index.html)'
+                              ' to get more information about the decoders'
+               },
+        1505: {'message': 'Invalid relative directory, '
+               'a decoder_dir tag must be declared in ossec.conf ruleset section.',
+               'remediation': f'Please, visit the official documentation'
+                               '(https://documentation.wazuh.com/'
+                              f'{DOCU_VERSION}/user-manual/reference/ossec-conf/index.html)'
+                              ' to get more information about the decoders'
+               },
+        1506: {'message': 'Uploading or deleting default decoders is not allowed.',
+               'remediation': f'Please, visit the official documentation'
+                               '(https://documentation.wazuh.com/'
```

Comment:
```suggestion
        1506: {'message': 'Updating or deleting default decoders is not allowed.',
               'remediation': f'Please, visit the official documentation'
                               '(https://documentation.wazuh.com/'
```

Better redirect to `/user-manual/reference/ossec-conf/ruleset.html` section of the documentation instead of `/user-manual/reference/ossec-conf/index.html`